README.md

```diff
@@ -46,7 +46,7 @@ Deep Renewal Processes           | Global       | Univariate               | RNN
 GPForecaster                     | Global       | Univariate               | MLP, Gaussian process | [MXNet](src/gluonts/model/gp_forecaster/_estimator.py) | -
 MQ-CNN                           | Global       | Univariate               | CNN encoder, MLP decoder | [MXNet](src/gluonts/model/seq2seq/_mq_dnn_estimator.py) | [paper](https://arxiv.org/abs/1711.11053)
 MQ-RNN                           | Global       | Univariate               | RNN encoder, MLP encoder | [MXNet](src/gluonts/model/seq2seq/_mq_dnn_estimator.py) | [paper](https://arxiv.org/abs/1711.11053)
-N-BEATS                          | Global       | Univariate               | MLP, residual links | [MXNet](https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/model/n_beats/_estimator.py) | [paper](https://openreview.net/forum?id=r1ecqn4YwB)
+N-BEATS                          | Global       | Univariate               | MLP, residual links | [MXNet](https://github.com/awslabs/gluon-ts/blob/dev/src/gluonts/model/n_beats/_estimator.py) | [paper](https://openreview.net/forum?id=r1ecqn4YwB)
```

Comment:
I think this should be a relative reference just like the others, can you fix it since you're at it?